src/test/test_pidfile.sh

```diff
@@ -33,6 +33,7 @@ function TEST_without_pidfile() {
         --mkfs \
         --mon-data=$data \
         --run-dir=$dir || return 1
+    sleep 1
```

Comment:
i don't think this sleep des anything.. --mkfs runs and then exits, so nothing is running at this point